taca/nanopore/ONT_run_classes.py

```diff
@@ -68,9 +68,10 @@ def __init__(self, run_abspath: str):
                 self.rsync_options[k] = None
 
         # Get transfer details, depending on run type and instrument
-        self.transfer_details = CONFIG["nanopore_analysis"]["run_types"][self.run_type][
-            "instruments"
-        ][self.instrument]
+        if hasattr(self, "run_type"):
```

Comment:
I would change this to set `self.run_type = None  # Will be set in subclasses` and `if self.run_type is not None`. I think it's easier to understand. What do you think?

Reply:
[fe0f44e](https://github.com/SciLifeLab/TACA/pull/410/commits/fe0f44e50509e8a78d80f3d162dae9026b669083)